worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/StorageManager.scala

```diff
@@ -603,6 +610,16 @@ final private[worker] class StorageManager(conf: CelebornConf, workerSource: Abs
     false
   }
 
+  def persistShuffle(shuffleKey: String, sync: Boolean = false): Unit = {
```

Comment:
What's the specific reason that `sync` is set differently in normal persist and final persist?